pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -686,6 +668,15 @@ func allocatableResources(memory, cpu string) v1.ResourceList {
 
 // DaemonSets should not place onto nodes with insufficient free resource
 func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
+	enabled := utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods)
+	// Rollback feature gate.
+	defer func() {
+		if enabled {
+			utilfeature.DefaultFeatureGate.Set("ScheduleDaemonSetPods=true")
+		}
+	}()
+
+	utilfeature.DefaultFeatureGate.Set("ScheduleDaemonSetPods=false")
```

Comment:
Could we add a comment about why the feature gate is disabled in this test?

Reply:
Sure.